test/divide_segment.test.js

```diff
@@ -74,8 +74,8 @@ tap.test('divide segments', function(t) {
   });
 
   t.test('possible intersections on 2 polygons', function(t) {
-    var s = subject.geometry.coordinates;
-    var c = clipping.geometry.coordinates;
+    var s = [subject.geometry.coordinates];
+    var c = [clipping.geometry.coordinates];
```

Comment:
@rowanwins Why are the test geometry changing? Aren't you handling (Multi)Polygon geometry [here](https://github.com/w8r/martinez/pull/18/files#diff-1fdf421c05c1140f6d71444ea2b27638R630)?
Adding that extra Array means the input is a MultiPolygon and not a Polygon.
Same comment for `index.test.js`